docs/toc.md

```diff
@@ -2,6 +2,51 @@
 
 <!-- Start of .NET Platform Guide -->
 # [.NET Platform Guide](standard/)
```

Comment:
I think you need to change this to standard/index.md and remove the toc.md file from that folder. the TOC now is all broken now on stage